executor/hnswpsql.py

```diff
@@ -0,0 +1,189 @@
+import copy
+from datetime import datetime
+from typing import Optional, Tuple, Dict
+
+import numpy as np
+from jina import Executor, requests, DocumentArray, Document
+from jina.logging.logger import JinaLogger
+
+from .hnswlib_searcher import HnswlibSearcher
+from .postgres_indexer import PostgreSQLStorage
+
+
+class HNSWPostgresIndexer(Executor):
+    """
+    TODO
+
+    """
+
+    def __init__(self,
+                 total_shards: Optional[int] = None,
+                 startup_sync: bool = True,
+                 rebuild: bool = True,
+                 **kwargs):
+        """
+        :param startup_sync: whether to sync from PSQL into HNSW on start-up
+        :param total_shards: the total nr of shards that this shard is part of.
+        :param rebuild: assume all sync operations should completely rebuild HNSW
+
+            NOTE: This is REQUIRED in k8s, since there `runtime_args.parallel` is
```

Comment:
```suggestion
            NOTE: `total_shards` is REQUIRED in k8s, since there `runtime_args.parallel` is
```

Reply:
Is this the right translation of "This"?

Reply:
Ah yes. Good catch, thx